source/server/server.cc

```diff
@@ -51,6 +51,15 @@ InstanceImpl::InstanceImpl(Options& options, Network::Address::InstanceConstShar
       dns_resolver_(dispatcher_->createDnsResolver({})),
       access_log_manager_(*api_, *dispatcher_, access_log_lock, store) {
 
+  if (!options.logPath().empty()) {
+    try {
+      Logger::Registry::getSink()->logToFile(options.logPath(), access_log_manager_);
+    } catch (const EnvoyException& e) {
+      PANIC(
```

Comment:
Sorry one other thing. Is it possible to get test coverage of this path? Either via death test or switch to exception which would be caught by main IIRC? (And can be tested via server_test).

Reply:
Fixed.